### PR TITLE
Ad suggestions form functionality

### PR DESCRIPTION
**Pull request by abbyjoydemarest**
-takes data and puts in data base
-gives bra suggestion for frame
-sends user to results page that gives bra size and bra suggestion(maybe we can add confetti to this later)
-We will need to fix the url for the results page right now it lists suggestion-form twice in the path

### CODE REVIEW

**Comment by abbyjoydemarest** [comment]
You will have to MIGRATE after merge!

**Comment by yyapuncich** [comment]
HEY AWESOME it's working.
There are merge conflicts with master you can resolve before we merge in? 
Also the suggestions, we know, need work in the wording:

Screen Shot 2019-04-19 at 10 22 15 PM

File: core/forms.py [hunk 1]
```diff
@@ -17,4 +17,4 @@ class SuggestionForm(forms.ModelForm):
     """
     class Meta:
         model = Suggestion
-        fields = ('breast_symmetry', 'breast_tissue', 'breast_placement', 'bra_wire')
+        fields = ('breast_symmetry', 'breast_tissue', 'breast_placement', 'bra_padding', 'bra_wire',)
```
Comment (yyapuncich) [inline core/forms.py]: Are we going to take out breast symmetry? 
And rename breast_tissue to shape?

**Review by yyapuncich — CHANGES_REQUESTED** [review]
looks awesome. I'm so excited to have these working@@@!!!!!

**Comment by abbyjoydemarest** [comment]
> HEY AWESOME it's working.
> There are merge conflicts with master you can resolve before we merge in?
> Also the suggestions, we know, need work in the wording:
> 
> Screen Shot 2019-04-19 at 10 22 15 PM

Yeah I totally agree with wording. Just wanted to get some content out there. Anything can be adjusted!

File: core/forms.py (hunk 1 above)
Comment (abbyjoydemarest) [reply core/forms.py]: I changed breast_tissue to breast_shape. Do we need the symmetry stuff for the bra suggestion ajax?

**Review by yyapuncich — APPROVED** [review]

**Pull request merged**